src/bundle/Resources/public/js/scripts/button.trigger.js

```diff
@@ -1,17 +1,19 @@
-document.addEventListener(
-    'DOMContentLoaded',
-    () => {
-        const buttons = document.querySelectorAll('.ibexa-btn--trigger');
-        const trigger = (event) => {
-            event.preventDefault();
+(function (global, doc) {
+    doc.addEventListener(
+        'DOMContentLoaded',
+        () => {
+            const buttons = doc.querySelectorAll('.ibexa-btn--trigger');
+            const trigger = (event) => {
```

Comment:
What do you think to move this function out of listener and rename it to e.g `triggerElement`?

Reply:
I just changed indentation here, didn't indent to refactor whole file :)